pkg/provisioningv2/rke2/installer/server.go

```diff
@@ -24,5 +24,8 @@ func (s *handler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		return
 	}
 	rw.Header().Set("Content-Type", "text/plain")
-	rw.Write(content)
+	_, err = rw.Write(content)
+	if err != nil {
+		return
+	}
```

Comment:
We are essentially ignoring the error and trying to make go-vet happy. This is the proper way to do that.
```suggestion
_, _ = rw.Write(content)
```